examples/hpc/dla-driver-ffi/src/lib.rs

```diff
@@ -459,28 +436,20 @@ pub unsafe extern "C" fn dla_tvm_qnn_conv2d(
         None,
     );
 
+    let input_order_string = unsafe { CStr::from_ptr(input_order).to_str().unwrap_unchecked() };
```

Comment:
I guess TVM is always *guaranteed* to give valid order strings. This would be a great way inject a buffer underrun/overflow through the FFI :) But we don't much care about those for this so I think this is fine and efficient.

Reply:
Yes this string is hard-coded to Headsail TVM BYOC backend. One could use this for malicious purposes but this is fast and easy for us. When NVIDIA buys us, I will fix this.